packages/babel-plugin/src/index.tsx

```diff
@@ -1,185 +1,2 @@
-            opts: { importReact: shouldImportReact = true },
-          } = state;
-
-          if (shouldImportReact && !path.scope.getBinding('React')) {
-            // React is missing - add it in at the last moment!
-            path.unshiftContainer('body', template.ast(`import * as React from 'react'`));
-          }
-
-          if (state.compiledImports.styled && !path.scope.getBinding('forwardRef')) {
-            // forwardRef is missing - add it in at the last moment!
-            path.unshiftContainer('body', template.ast(`import { forwardRef } from 'react'`));
-          }
-
-          const version = process.env.TEST_PKG_VERSION || pkgJson.version;
-
-          path.addComment(
-            'leading',
-            ` ${parseFilename(state.filename)} generated by ${pkgJson.name} v${version} `
-          );
-
-          // Add a line break the comment
-          path.unshiftContainer('body', t.noop());
-        },
-      },
-      ImportDeclaration(path, state) {
-        if (path.node.source.value !== '@compiled/react') {
-          return;
-        }
-
-        // The presence of the module enables CSS prop
-        state.compiledImports = {};
-
-        // Go through each import and enable each found API
-        path.get('specifiers').forEach((specifier) => {
-          if (!state.compiledImports || !specifier.isImportSpecifier()) {
-            // Bail out early
-            return;
-          }
-
-          (['styled', 'ClassNames'] as const).forEach((apiName) => {
-            if (
-              state.compiledImports &&
-              t.isIdentifier(specifier.node?.imported) &&
-              specifier.node?.imported.name === apiName
-            ) {
-              // Enable the API with the local name
-              state.compiledImports[apiName] = specifier.node.local.name;
-
-              // Remove specifier
-              specifier.remove();
-            }
-          });
-        });
-
-        appendRuntimeImports(path);
-
-        if (path.node.specifiers.length === 0) {
-          // No more imports - remove the whole lot!
-          path.remove();
-        }
-      },
-      TaggedTemplateExpression(path, state) {
-        if (!state.compiledImports?.styled) {
-          return;
-        }
-
-        visitStyledPath(path, { state, parentPath: path });
-      },
-      CallExpression(path, state) {
-        if (!state.compiledImports) {
-          return;
-        }
-
-        visitStyledPath(path, { state, parentPath: path });
-      },
-      JSXElement(path, state) {
-        if (!state.compiledImports?.ClassNames) {
-          return;
-        }
-
-        visitClassNamesPath(path, { state, parentPath: path });
-      },
-      JSXOpeningElement(path, state) {
-        if (!state.compiledImports) {
-          return;
-        }
-
-        visitCssPropPath(path, { state, parentPath: path });
-      },
-    },
-  };
-});
+export { default } from './babel-plugin';
+export { transform } from './transform';
```

Comment:
so we can transform directly if needed. currently using this in webpack loader - however it might not be needed in parcel. maybe.